dashboard/app/controllers/home_controller.rb

```diff
@@ -34,6 +34,23 @@ def health_check
 
   GALLERY_PER_PAGE = 5
   def index
+    if request.cookies['pm'] == 'new_header'
+      redirect_to '/home'
+    else
+      init_homepage
+    end
+  end
+
+  def home
+    if request.cookies['pm'] != 'new_header'
+      redirect_to '/'
+    else
+      init_homepage
+      render 'home/index'
+    end
+  end
+
+  def init_homepage
```

Comment:
I'll defer to @aoby's expertise here.

This is an example of where I feel like things would be much easier to follow if we made it explicit where we are rendering things, but I know we've said as a team we prefer "the rails way" of leaving things implicit where possible.